src/Shared/Microsoft.DotNet.Scaffolding.Shared/T4Templating/TextTemplatingEngineHost.cs

```diff
@@ -0,0 +1,275 @@
+        ///     any release. You should only use it directly in your code with extreme caution and knowing that
+        ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+        /// </summary>
+        public virtual IList<string> StandardAssemblyReferences { get; } = new string[]
+        {
+            typeof(ITextTemplatingEngineHost).Assembly.Location,
+            typeof(CompilerErrorCollection).Assembly.Location
+        };
+
+        /// <summary>
+        ///     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
+        ///     the same compatibility standards as public APIs. It may be changed or removed without notice in
+        ///     any release. You should only use it directly in your code with extreme caution and knowing that
+        ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+        /// </summary>
+        public virtual IList<string> StandardImports { get; } = new[]
+        {
+            "System"
+        };
+
+        /// <summary>
+        ///     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
+        ///     the same compatibility standards as public APIs. It may be changed or removed without notice in
+        ///     any release. You should only use it directly in your code with extreme caution and knowing that
+        ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+        /// </summary>
+        public virtual string TemplateFile { get; set; }
+
+        /// <summary>
+        ///     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
+        ///     the same compatibility standards as public APIs. It may be changed or removed without notice in
+        ///     any release. You should only use it directly in your code with extreme caution and knowing that
+        ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+        /// </summary>
+        public virtual string Extension
+            => _extension ?? ".cs";
+
+        /// <summary>
+        ///     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
+        ///     the same compatibility standards as public APIs. It may be changed or removed without notice in
+        ///     any release. You should only use it directly in your code with extreme caution and knowing that
+        ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+        /// </summary>
+        public virtual CompilerErrorCollection Errors
+            => _errors == null ? new CompilerErrorCollection() : _errors;
```

Comment:
>_errors == null ? new CompilerErrorCollection() : _errors; [](http://example.com/codeflow?start=15&length=58)

same trick here if you decide to apply it

Reply:
had to update the language but done, tyty